doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -887,6 +887,77 @@ However elimination to `Type` or to a polymorphic sort with `s := Prop` is allow
         : Type@{s|max(u,v)}
         := pair { pr1 : A; pr2 : B pr1 }.
 
+Explicit Sorts
+---------------
+
+Similar to universes, fresh global sorts can be declared with the :cmd:`Sort`.
+
+.. cmd:: Sort {+ @ident }
+         Sorts {+ @ident }
+
+   In the monomorphic case, declares new global sort qualities
+   with the given names.  Global quality names live in their own namespace.
+   Inside sections, the command respects the `Universe Polymorphism` flag,
+   either set globally or locally through the :attr:`universes(polymorphic)` attribute (or
+   the ``Polymorphic`` legacy attribute), meaning the sort
+   quantification will be discharged for each section definition
+   independently. Polymorphic sort qualities are forbidden outside sections.
+
+  .. exn:: Polymorphic sorts can only be declared inside sections, use #[universe(polymorphic=no)] Sort in order to declare a global sort.
+
+    Generated when a :cmd:`Sort` command is issued outside a section with universe polymorphism set.
+    Either scope the :cmd:`Sort` with an enclosing :cmd:`Section` or be explicit about creating a global sort
+    by turning universe polymorphism off.
+
+  .. exn:: Cannot declare global sort qualities inside module types.
+
+    The :cmd:`Sort` command is not supported inside module types.
+
+.. cmd:: Print Sorts
+   :name: Print Sorts
```

Comment:
This does not seem to work. And even if it did, it would be pointless since the specified name is identical to the command.

```suggestion
```